docs/medical-api/more-info/webhooks.mdx

```diff
@@ -36,23 +47,25 @@ Below is an example payload you could send in the request body of one of those e
 
 These are messages you can expect to receive in the following scenarios:
 
-1. When [queried documents](/medical-api/api-reference/document/start-document-query) have completed downloading (`type` for a patient in
-   the message will be `document-download`, and at this point you'll be able to [download the raw files](/medical-api/api-reference/document/get-document));
+1. When [queried documents](/medical-api/api-reference/document/start-document-query) have completed
+   downloading, the message `type` will be `medical.document-download`, and at this point 
+   you'll be able to [download the raw files](/medical-api/api-reference/document/get-document);
+2. Then, if the downloaded documents contained C-CDA/XML files, when the conversion to FHIR has 
+   completed, the message `type` will be `medical.document-conversion`, and at this point
+   you'll be able to query for [patient consolidated data](#patient-consolidated-data) in 
+   FHIR-compliant format.
```

Comment:
for consistency - let's link to the endpoint they can use to do the query as per above, not the webhook message